src/auth-service/utils/create-user.js

```diff
@@ -1064,6 +1107,80 @@ const join = {
       };
     }
   },
+
+  deleteMobileUserData: async (request) => {
+    try {
+      const { body } = request;
+      let { uid } = body;
+      let { ct } = body;
+      const userRecord = await admin.auth().getUser(uid);
+      
+      //get creation time and compare with ct
+      let creationTime = userRecord.metadata.creationTime;
+      creationTime = creationTime.replace(/\D/g, "");
+      if (creationTime !== ct) {
+        return {
+          success: false,
+          message: "Invalid request",
+          status: httpStatus.BAD_REQUEST,
+          errors: { message: "Invalid request" },
+        };
+      }
+
+      try {
+        await getAuth().deleteUser(uid);
+        const collectionList = [
+          constants.FIREBASE_COLLECTION_KYA,
```

Comment:
Hi @BenjaminSsempala , I do hope that these constants were added to the constants file and subsequently to the secrets if need be? Please clarify on this. Thanks!

Reply:
Yes @Baalmart , these were already there in both the constants and corresponding secrets file.

Reply:
Thanks @BenjaminSsempala , please share them with me on Slack so that I can also update my local ENV